linux/power_operation_scripts/set_power_script.yml

```diff
@@ -0,0 +1,111 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Set and check soft power operation
+# # Parameters:
+#   power_script_op: power script operation
+#   power_script_type: default or custom
+
+- name: "Start to test {{ power_script_type }} {{ power_script_op }} script"
+  ansible.builtin.debug:
+    msg:
+      - "Start to test {{ power_script_type }} {{ power_script_op }} script"
+
+- name: "Set {{ power_script_type }} script for {{ power_script_op }} script"
```

Comment:
`Set default script for power script`... The task name doesn't read smoothly. Maybe you can change to `Set {{ power_script_type }} {{ power_script_op }} script"`